File: Assets/Scripts/CSharp/FaceManager.cs
using UnityEngine;
using System.Collections;

public class FaceManager : MonoBehaviour 
{
	public GameObject GirlPre;
	public GameObject GirlPre2;
	public GameObject BoyPre;
	public GameObject BoyPre2;
	public int Gender;
	
	void Start () 
	{
		Gender = PlayerPrefs.GetInt("Gender");
		
		if(Gender == 0)
		{
			BoyPre.SetActive(true);
			BoyPre2.SetActive(true);
		}
		else if(Gender == 1)
		{
			GirlPre.SetActive(true);
			GirlPre2.SetActive(true);
		}
	}
	
	void Update () 
	{
	
	}
}

File: Assets/Scripts/FaceManager.js
#pragma strict
var GirlPre:GameObject;
var GirlPre2:GameObject;
var BoyPre:GameObject;
var BoyPre2:GameObject;
var Gender:int;
function Start () {
	Gender = PlayerPrefs.GetInt("Gender");
	if(Gender == 0)
	{
		BoyPre.SetActive(true);
		BoyPre2.SetActive(true);
	}
	else if(Gender == 1)
	{
		GirlPre.SetActive(true);
		GirlPre2.SetActive(true);
	}
}

function Update () {

}